<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>190</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sitka</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="primary00ColorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Primary Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondary00ColorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Secondary Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="primary01RLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>primary01RLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="secondary01RLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>secondary01RLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="primary02GLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>primary02GLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="secondary02GLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>secondary02GLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="primary03BLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>primary03BLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="secondary03BLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>secondary03BLineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="primary01RLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>76</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondary01RLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>255</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="primary02GLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>114</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="primary03BLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>29</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondary02GLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>255</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondary03BLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>225</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>460</width>
     <height>100</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="title">
    <string>GUI Color Scheme</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>primary00ColorLabel</zorder>
  <zorder>secondary00ColorLabel</zorder>
  <zorder>primary01RLabel</zorder>
  <zorder>secondary01RLabel</zorder>
  <zorder>primary02GLabel</zorder>
  <zorder>secondary02GLabel</zorder>
  <zorder>primary03BLabel</zorder>
  <zorder>secondary03BLabel</zorder>
  <zorder>acceptButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>primary01RLineEdit</zorder>
  <zorder>secondary01RLineEdit</zorder>
  <zorder>primary02GLineEdit</zorder>
  <zorder>primary03BLineEdit</zorder>
  <zorder>secondary02GLineEdit</zorder>
  <zorder>secondary03BLineEdit</zorder>
 </widget>
 <tabstops>
  <tabstop>primary01RLineEdit</tabstop>
  <tabstop>primary02GLineEdit</tabstop>
  <tabstop>primary03BLineEdit</tabstop>
  <tabstop>secondary01RLineEdit</tabstop>
  <tabstop>secondary02GLineEdit</tabstop>
  <tabstop>secondary03BLineEdit</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
